Vagrantfile

```diff
@@ -0,0 +1,19 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure('2') do |config|
+	config.vm.box = "bento/ubuntu-15.04"
+        
+        config.vm.provider "virtualbox" do |vb|
+            vb.name = "reprozip-examples"
+            # vb.gui = true
+        end
+
+        config.vm.network "forwarded_port", guest: 8000, host: 8111
```

Comment:
I wanted to be consistent with the original repository ([sdp_curricula](https://github.com/fchirigati/sdp_curricula)).